extension/js/background_page/background_page.ts

```diff
@@ -60,8 +59,6 @@ console.info('background_process.js starting');
   BrowserMsg.bgAddListener('storeGlobalSet', (r: Bm.StoreGlobalSet) => GlobalStore.set(r.values));
   BrowserMsg.bgAddListener('storeAcctGet', (r: Bm.StoreAcctGet) => AcctStore.get(r.acctEmail, r.keys));
   BrowserMsg.bgAddListener('storeAcctSet', (r: Bm.StoreAcctSet) => AcctStore.set(r.acctEmail, r.values));
-  BrowserMsg.bgAddListener('processAndStoreKeysFromEkmLocally', processAndStoreKeysFromEkmLocally);
-  BrowserMsg.bgAddListener('getLocalKeyExpiration', getLocalKeyExpiration);
 
   // todo - when https://github.com/FlowCrypt/flowcrypt-browser/issues/2560
   //   is fixed, this can be moved to the gmail content script, and some may be removed
```

Comment:
Also `BrowserMsg.addPgpListeners();` could be removed from below, and added to content script message listener instead.

Then pgp_block can be updated to call content script instead of background script.

Reply:
Should I do this in this PR?
I thought after #5022 we may no longer need some of these messages, e.g. `pgpMsgDecrypt` or `pgpMsgVerifyDetached`

Reply:
i see. then it doesn't have to be in this pr.